unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -746,8 +747,12 @@ public void SaveAvatar(Texture2D face256Snapshot, Texture2D bodySnapshot)
 
         WebInterface.SendSaveAvatar(avatarModel, face256Snapshot, bodySnapshot, DataStore.i.common.isSignUpFlow.Get());
         userProfile.OverrideAvatar(avatarModel, face256Snapshot);
+        
         if (DataStore.i.common.isSignUpFlow.Get())
+        {
             DataStore.i.HUDs.signupVisible.Set(true);
+            NewUserExperienceAnalytics.AvatarEditSuccessNux();
```

Comment:
In this script we already have `internal IAnalytics analytics` variable. Do you think we need NewUserExperienceAnalytics class at all?

Reply:
Yes, I think it makes sense to have a specialized wrapper for it same as we do in other places i.e. SocialAnalytics or GenericAnalytics. This helps to easily find all of the events related to a particular feature scope.